### DISCUSSION TITLE
Modulation in scan modus not works

### DISCUSSION THREAD

**Discussion by buentAir** [rtl-airband/RTLSDR-Airband#285] [Q&A] [answered]
Hi

If I set the modulation for the individual channels, this error is output:
Configuration error: invalid parameter type: devices.[3].channels.[0].modulation

Setup:
      freqs = ( 119.305, 123.45, 130.705, 130.8, 252.5, 255.825, 266.775, 273.95, 302.55, 307.3, 358.9 );
      modulation = ( "am", "am", "am", "am", "nfm", "nfm", "nfm", "nfm", "nfm", "nfm", "nfm" );

freqs and modulation, both are 11 entries.

If I set the modulation to the standard, the setup works:
modulation = "am";

Yes, narrow FM demodulation support is enabled. :-)

Regards
Mathias

**Comment by szpajder** [comment, marked as answer]
That's an error in the docs; the correct name of the per-frequency setting is `modulations`.

I've fixed the wiki page. Thanks for pointing this out.

**Comment by buentAir** [comment]
Sometimes so easy :-)
I thank you!
